apiary.apib

```diff
@@ -98,18 +98,17 @@ Besides `/suggester` and `/search` endpoints, everything is accessible from `loc
 
         [
           {
-            "acceptedTime": "2018-06-28T17:49:01.793Z",
-            "message": {
-              "tags": ["main"],
-              "cssClass": "cssClass",
-              "text": "test message",
-              "duration": "PT10M"
-            },
-            "expirationTime": "2018-06-28T17:59:01.793Z",
-            "expired":false
+            "expired": false,
+            "created": "2019-01-23 20:39:31 CET",
+            "tags": [
+              "foo"
```

Comment:
the tag should be main, shouldn't it?

Reply:
Well, the tag can be arbitrary. That said it makes the documentation a bit friendlier to make it appear like it was produced by the POST message above.

Reply:
ah I see this "foo" is not connected to the "main" above.